### ISSUE TITLE
Instance won't connect successfully

### ISSUE THREAD

**Issue by Signum** [iobroker-community-adapters/ioBroker.homeconnect#39]
Just bought a new dishwasher today. Connection works from my smartphone. Developer account created. Application created. I tried the procedure multiple times and even deleted my developer account and created it from scratch. But I won't be let in.

Adapter version v1.0.1

Settings:
App-Email: homeconnect@christoph-…
App-Password: the password for my account
Client-ID: The ID I got from the developer portal

Application:
![grafik](https://user-images.githubusercontent.com/36562/172024945-8e1d6bdc-eadf-4ae2-881b-1715591330a1.png)
(ID truncated on purpose)

Various error messages I find in the logs:

```
The  authorization request is still pending as the end-user hasn't yet  completed the user interaction steps
Error: Request failed with status code 400
Please  check username and password check manually here:  https://api.home-connect.com/security/oauth/device_verify?user_code=J35P-…

No verification_uri_complete in device_authorization
{"error":"unauthorized_client","error_description":"request rejected by client authorization authority (developer portal)"}
Error: Request failed with status code 400
```

I'm out of ideas here. Any hint where I could look?

**Comment by TA2k** [comment]
Please recreate the client id and test the username passwort with the url in the log

**Comment by Signum** [comment]
Thanks. Solved. I used the URL from the logs, was asked to authenticate and the "ioBroker" application was confirmed. The instance works and has already found my dishwasher.

I could swear I tried to use that URL before and was asked to "enter the code shown on the device" which didn't make sense to me. But re-creating the application and using the URL worked instantly.

Oh, and you deserve a medal for that quick support. :) Thank you so much.
